device/iot-device-client/pom.xml

```diff
@@ -107,6 +107,11 @@
             <artifactId>slf4j-api</artifactId>
             <version>1.7.25</version>
         </dependency>
+        <dependency>
+            <groupId>commons-httpclient</groupId>
+            <artifactId>commons-httpclient</artifactId>
+            <version>3.1</version>
```

Comment:
How huge is this dependency>

Reply:
Does it work on android?

Reply:
300kb, so it's not that bad. The gate will let us know if android will be an issue, but I don't expect it will be